tests/conftest.py

```diff
@@ -778,11 +792,24 @@ def setup_for_snippet(self, snippet, autostd=True, install_v2_modules: List[Loca
         :param install_v2_modules: Indicates which V2 modules should be installed in the compiler venv
         """
         self.setup_for_snippet_external(snippet)
+        loader.unload_inmanta_plugins()
+        loader.PluginModuleFinder.reset()
         project = Project(self.project_dir, autostd=autostd)
         Project.set(project)
         project.use_virtual_env()
+        self._patch_process_env(project)
         self._install_v2_modules(project, install_v2_modules)
 
+    def _patch_process_env(self, project: Project) -> None:
```

Comment:
Why is this required (or rather, why is it required now and not in the past)

Reply:
Because with modules v2 we install packages in the wrapping developer environment (not the compiler environment) through `inmanta.env.ProcessEnv`. If we're running tests we don't actually want to install anything in the developer's environment, so we need to mock the wrapping environment.